tool/gravity/cli/utils.go

```diff
@@ -293,25 +293,36 @@ func loadRPCCredentials(ctx context.Context, addr, token string) (*rpcserver.Cre
 	return creds, nil
 }
 
-// hasFlagInArgs returns true if the specified flag has been found
-// in args.
+// updateFlagsInArgs returns the list of flags missing from args.
+// flags specifies the flags to search for - if a flag is missing from args,
+// it is returned as part of the result.
 // parser specifies the command line parser for args
-func hasFlagInArgs(flag string, args []string, parser ArgsParser) (ok bool, err error) {
+func updateFlagsInArgs(flags []flag, args []string, parser ArgsParser) (flagsToAdd []string, err error) {
```

Comment:
Nit: Wouldn't something like `getMissingFlags` be a more appropriate name for this function (judging by godoc)?